src/lib/decentraland/ByteBuffer/index.ts

```diff
@@ -148,104 +148,104 @@ export class ReadWriteByteBuffer implements ByteBuffer {
+    this.#view.setUint16(o, value, true)
   }
   writeUint32(value: number): void {
     const o = this.#woAdd(4)
-    this.#view.setUint32(o, value)
+    this.#view.setUint32(o, value, true)
   }
   writeUint64(value: bigint): void {
     const o = this.#woAdd(8)
-    this.#view.setBigUint64(o, value)
+    this.#view.setBigUint64(o, value, true)
   }
   // DataView Proxy
   getFloat32(offset: number): number {
-    return this.#view.getFloat32(offset)
+    return this.#view.getFloat32(offset, true)
   }
   getFloat64(offset: number): number {
-    return this.#view.getFloat64(offset)
+    return this.#view.getFloat64(offset, true)
   }
   getInt8(offset: number): number {
     return this.#view.getInt8(offset)
   }
   getInt16(offset: number): number {
-    return this.#view.getInt16(offset)
+    return this.#view.getInt16(offset, true)
   }
   getInt32(offset: number): number {
-    return this.#view.getInt32(offset)
+    return this.#view.getInt32(offset, true)
   }
   getInt64(offset: number): bigint {
-    return this.#view.getBigInt64(offset)
+    return this.#view.getBigInt64(offset, true)
   }
   getUint8(offset: number): number {
     return this.#view.getUint8(offset)
   }
   getUint16(offset: number): number {
-    return this.#view.getUint16(offset)
+    return this.#view.getUint16(offset, true)
   }
   getUint32(offset: number): number {
-    return this.#view.getUint32(offset) >>> 0
+    return this.#view.getUint32(offset, true) >>> 0
   }
   getUint64(offset: number): bigint {
-    return this.#view.getBigUint64(offset)
+    return this.#view.getBigUint64(offset, true)
   }
   setFloat32(offset: number, value: number): void {
-    this.#view.setFloat32(offset, value)
+    this.#view.setFloat32(offset, value, true)
   }
   setFloat64(offset: number, value: number): void {
-    this.#view.setFloat64(offset, value)
+    this.#view.setFloat64(offset, value, true)
   }
   setInt8(offset: number, value: number): void {
     this.#view.setInt8(offset, value)
   }
   setInt16(offset: number, value: number): void {
-    this.#view.setInt16(offset, value)
+    this.#view.setInt16(offset, value, true)
   }
   setInt32(offset: number, value: number): void {
-    this.#view.setInt32(offset, value)
+    this.#view.setInt32(offset, value, true)
   }
   setInt64(offset: number, value: bigint): void {
-    this.#view.setBigInt64(offset, value)
+    this.#view.setBigInt64(offset, value, true)
   }
   setUint8(offset: number, value: number): void {
     this.#view.setUint8(offset, value)
   }
   setUint16(offset: number, value: number): void {
-    this.#view.setUint16(offset, value)
+    this.#view.setUint16(offset, value, true)
   }
   setUint32(offset: number, value: number): void {
-    this.#view.setUint32(offset, value)
+    this.#view.setUint32(offset, value, true)
   }
   setUint64(offset: number, value: bigint): void {
-    this.#view.setBigUint64(offset, value)
+    this.#view.setBigUint64(offset, value, true)
```

Comment:
_Comment suggestion_
Now that this class explicitly works with little-endian, it could be helpful to specify that in the comment above the class definition ("ByteBuffer is a wrapper of DataView...")